### PR TITLE
150 create docs homepage

### PR DESCRIPTION
**Pull request by ghost**
### What does this PR fix/introduce?
Enhance the docs landing page to give some more information about how to use the documentation

### Checklist
(Delete any that aren't relevant)

- [x] Docs are successfully building - `yarn install && yarn run build`.
- [x] For new **internal** links I used *relative file paths* (with .md extension) - e.g. `../../faq/faq-general.md` - instead of introducing *absolute file path*, or *relative/absolute URL*.
- [x] All external links have been verified with `yarn run check:externals`.
- [x] My changes follow the [Casper docs style guidelines](https://docs.casperlabs.io/workflow/contribute/).
- [x] All technical procedures have been tested (if you want help with this, mention it in [Reviewers](#reviewers)).
- [x] If structural changes are introduced (not just content changes), cross-broswer testing has been completed.

### Reviewers
// Tag whoever needs to review. If you're not sure, leave blank.

### CODE REVIEW

**Comment by ghost** [comment]
Not yet ready for review - closing to avoid confusion

File: source/docs/casper/index.md [hunk 1]
```diff
@@ -5,90 +5,47 @@ slug: /
 tags: ["Must read", "Legal", "Introduction", "CasperLabs", "CSPR"]
 ---
 
-# Welcome to the Casper Network
+# Welcome to the Casper Network Documentation
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md:8]:
```suggestion
# Welcome to the Casper Network Docs
```

I think this way it's not going to wrap the heading into two lines on most devices

File: source/docs/casper/index.md [hunk 2]
```diff
@@ -5,90 +5,47 @@ slug: /
...
 
-This article provides an overview of the Casper project.
+This is the landing page for all Casper documentation.
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
```

I think it's already explained in the headline

File: source/docs/casper/index.md [hunk 3]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
```

File: source/docs/casper/index.md [hunk 4]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+# Get Started Here
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]: This heading is not necessary 

```suggestion
```

File: source/docs/casper/index.md [hunk 5]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+Read more about Casper [here](./about.md). For a discussion of the Casper design read [this article](./concepts/design/casper-design.md).
 
--   **Risks associated with abandonment / lack of success**: the Recipient understands and accepts that the creation of the CSPR and the development of the Casper blockchain as well as the CasperLabs Networks AG project may be abandoned for a number of reasons, including lack of interest from the public, lack of funding, lack of prospects (e.g. caused by competing projects) and legal, tax or regulatory considerations. The Recipient therefore understands that there is no assurance that, even if the CSPR/CasperLabs blockchain project is partially or fully developed and launched, the Recipient will receive any benefits through the CSPR held by it (if and when created and/or issued).
+# Using this documentation
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md:17]: This heading is not necessary 

```suggestion
```

File: source/docs/casper/index.md [hunk 6]
```diff
@@ -5,90 +5,47 @@ slug: /
...
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
```

File: source/docs/casper/index.md [hunk 7]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+ - [Running a node as an operator](./operators/index.md)
+ - [Design](./concepts/design/index.md)
+ - [Economics](./concepts/economics/index.md) 
+ - [Staking Guide](./concepts/economics/staking.md) 
+ 
+# Support 
+This documentation covers all of the technical aspects of Casper. However, if you can't find what you are looking for, or you simply want to explore the ecosystem, start with the [Support Page](./resources/support.md)
 
--   **Risks associated with forking**: the Recipient understands and accepts that hard and soft forks as well as similar events may, inter alia, lead to the creation of new or competing tokens to the CSPR, adversely affect the functionality, convertibility or transferability or result in a full or partial loss of units or reduction (including reduction to zero) of value of the Recipient’s CSPR (if and when created and/or issued).
+# [Legal](./disclaimer.md)
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]: This is going to be a link in the footer (footer is yet to come with changes to from MiracleDevs). 
```suggestion
```

File: source/docs/casper/concepts/index.md [hunk 8]
```diff
@@ -1,12 +1,31 @@
...
+Hashes are used everywhere in Casper for keys, addresses, packaging data nd more. Find a description of these hash types [here](./hash-types.md)
+### [Call Stacks](./callstack.md)  
+[This document](./callstack.md) contains a discussion of how Casper manages the calling of a contract, and some essential concepts to understand when running this kind of code. 
+### [Contracts and Session Code](./session-code.md)
+Read [this](./session-code.md) to understand the different kinds of code which execute on te Casper platform, the differences between them and why you would use one versus the other. 
+### [Dictionaries](./dictionaries.md)
+[Dictionaries](./dictionaries.md) are a primary construct for storing and retrieving data in Casper code. 
+### [Design](./design/index.md)
+An in-depth look at the high-level design and architecture of the Casper eco-system can be found [here](./design/index.md)
+### [Economics](./economics/index.md)
```
Comment (Adrian-Wrona) [inline source/docs/casper/concepts/index.md:27]: There is something wrong with this link or this page's slug.
When you click on it it takes you to http://localhost:3000/concepts/economics/concepts/economics/

File: source/docs/casper/disclaimer.md [hunk 9]
```diff
@@ -1,42 +1,61 @@
...
+This Whitepaper includes “forward-looking statements”, which are all statements other than statements of historical facts included in this Whitepaper. Words like “believe”, “anticipate”, “expect”, “project”, “estimate”, “predict”, “intend”, “target”, “assume”, “may”, “might”, “could”, “should”, “will” and similar expressions are intended to identify such forward-looking statements. Such forward-looking statements involve known and unknown risks, uncertainties and other factors, which may cause the actual functionality, performance or features of the Casper blockchain and/or CSPR to be materially different from any future functionality, performance or features expressed or implied by such forward-looking statements. Such forward-looking statements are based on numerous assumptions regarding the CasperLabs Networks AG’s and/or any of its affiliates’ present and future expectations regarding the development of the Casper blockchain and the associated software.
```
Comment (Adrian-Wrona) [inline source/docs/casper/disclaimer.md]: Are these changes to the quotation marks intentional?

File: source/docs/casper/index.md [hunk 10]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+# Using this documentation
 
--   **Risks associated with a loss of private key**: the Recipient understands and accepts that CSPR, if and when created and/or issued, will only be accessed by using a wallet technically compatible with CSPR and with a combination of the Recipient’s account information (address) and private key, seed or password. The Recipient understands and accepts that if its private key or password gets lost or stolen, the CSPR associated with the Recipient’s account (address) will be unrecoverable and will be permanently lost.
 
--   **Risks associated with wallets**: the Recipient understands and accepts that CasperLabs Networks AG or any of its affiliates, employees, partners or advisors are in no way responsible for the wallet to which any CSPR are transferred. The Recipient understands and agrees that it is solely responsible for the access and security of its wallet, for any security breach of its wallet and/or with any loss of CSPR resulting from its wallet service provider, including any termination of the service by the wallet provider and/or bankruptcy of the wallet provider.
+## Which Casper are you?
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]: Maybe something like this? I feel like there is something missing after 'Casper'

```suggestion
## What group do you belong to?
```

File: source/docs/casper/index.md [hunk 11]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+The Casper documentation is primarily organized around what you want to do with the Casper blockchain. We identify three loosely defined types of person interested in Casper.
 
--   **Risks associated with theft/hacks**: the Recipient understands and accepts that the smart contract, the website, the underlying software application and software platform (i.e. the Casper blockchain), during its development and after its launch, may be exposed to attacks by hackers or other individuals that could result in an inability to launch the Casper blockchain or the theft or loss of CSPR. Any such event could also result in the loss of financial and other support of the CasperLabs Networks AG project impacting the ability to develop the CasperLabs Networks AG project and Casper blockchain.
+  - ### Users
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]: I would align it to be the same as in concepts overview

```suggestion
  - ### [Users](./users/index.md)  
```

File: source/docs/casper/index.md [hunk 12]
```diff
@@ -5,90 +5,47 @@ slug: /
...
 
--   **Risks associated with theft/hacks**: the Recipient understands and accepts that the smart contract, the website, the underlying software application and software platform (i.e. the Casper blockchain), during its development and after its launch, may be exposed to attacks by hackers or other individuals that could result in an inability to launch the Casper blockchain or the theft or loss of CSPR. Any such event could also result in the loss of financial and other support of the CasperLabs Networks AG project impacting the ability to develop the CasperLabs Networks AG project and Casper blockchain.
+  - ### Users  
+Individuals who are interested in Casper from the point of view of personal interest, people using using an app based on Casper, people evaluating Casper, non-technical staff who wish to gain an understanding of the platform and its advantages and disadvantages. In this case, the [Concepts](./concepts/index.md) area is the best place to start, and will contain enough information for most casual users to gain an understanding of the platform.
+  - ### Developers
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
  - ### [Developers](./developers/index.md) 
```

File: source/docs/casper/index.md [hunk 13]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+  - ### Users  
+Individuals who are interested in Casper from the point of view of personal interest, people using using an app based on Casper, people evaluating Casper, non-technical staff who wish to gain an understanding of the platform and its advantages and disadvantages. In this case, the [Concepts](./concepts/index.md) area is the best place to start, and will contain enough information for most casual users to gain an understanding of the platform.
+  - ### Developers  
+The [Developers](./developers/index.md) area caters to software developers interested in build dApps on the Casper platform. This includes experienced developers with prior experience of Casper who are looking for specific information about certain features or API reference, as well as first-time developers looking to get their feet wet, learn the basics and create their first Casper dApp. For new developers, it is recommended to also check out the [concepts](./concepts/index.md) section first. It may be helpful to work through some of our [tutorials](./resources/tutorials/index.md). 
+  - ### Operators
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
  - ### [Operators](./operators/index.md)
```

File: source/docs/casper/index.md [hunk 14]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+  - ### Operators  
+The [Operators](./operators/index.md) section caters for people who want to run and administrate a Casper network. This includes technical staff, responsible for a production Casper network, developers who want to run a test node privately, individuals who run a Casper validator node.
 
--   **Risks associated with mining attacks**: the Recipient understands and accepts that, as with other cryptocurrencies and tokens, if and when launched, the Casper blockchain is susceptible to attacks relating to validators. Any successful attack presents a risk to the smart contract, expected proper execution and sequencing of transactions, and expected proper execution and sequencing of contract computations.
+## Other Sections
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
## Most Visited Pages
```

File: source/docs/casper/index.md [hunk 15]
```diff
@@ -5,90 +5,47 @@ slug: /
...
 
--   **Risks associated with a lack of consensus**: the Recipient understands and accepts that the network of validators will be ultimately in control of the genesis block and future blocks and that there is no warranty or assurance that the network of validators will perform their functions and reach proper consensus and allocate the CSPR to the Recipient as proposed by any terms. The Recipient further understands that a majority of the validators could agree at any point to make changes to the software and/or smart contracts and to run the new version of the software and/or smart contracts. Such a scenario could lead to the CSPR losing intrinsic value.
+- [FAQ](./faq/index.md)
+- [Resources](./resources/index.md)
+- [Tutorials](./resources/tutorials/index.md)
+- [Glossary](./concepts/glossary/index.md)
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]:
```suggestion
 - [Core Concepts of Casper](./concepts/index.md)
 - [Tutorial - Build your first dApp](./resources/tutorials/beginner/getting-started-tutorial.md)
 - [Running a node as an operator](./operators/index.md)
 - [Design](./concepts/design/index.md)
 - [Economics](./concepts/economics/index.md) 
 - [Staking Guide](./concepts/economics/staking.md) 
```

File: source/docs/casper/index.md [hunk 16]
```diff
@@ -5,90 +5,47 @@ slug: /
...
 
--   **Risks associated with liquidity of CSPR**: the Recipient understands and accepts that with regard to the CSPR, if and when created and/or issued, no market liquidity may be guaranteed and that the value of CSPR relative to other assets, including fiat currency and/or any other cryptocurrency (including any cryptocurrency used to acquire CSPR) over time may experience extreme volatility or depreciate in full (including to zero) resulting in loss that will be borne exclusively by the Recipient.
+# Most Visited Pages
+ - [Core Concepts of Casper](./concepts/index.md)
+ - [Tutorial - Build your first dApp](./resources/tutorials/beginner/getting-started-tutorial.md)
+ - [Running a node as an operator](./operators/index.md)
+ - [Design](./concepts/design/index.md)
+ - [Economics](./concepts/economics/index.md) 
+ - [Staking Guide](./concepts/economics/staking.md)
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md:33]:
```suggestion
## Other Sections
- [FAQ](./faq/index.md)
- [Resources](./resources/index.md)
- [Tutorials](./resources/tutorials/index.md)
- [Glossary](./concepts/glossary/index.md)
```

File: source/docs/casper/index.md [hunk 17]
```diff
@@ -5,90 +5,47 @@ slug: /
...
+# Most Visited Pages
+ - [Core Concepts of Casper](./concepts/index.md)
+ - [Tutorial - Build your first dApp](./resources/tutorials/beginner/getting-started-tutorial.md)
+ - [Running a node as an operator](./operators/index.md)
+ - [Design](./concepts/design/index.md)
+ - [Economics](./concepts/economics/index.md) 
+ - [Staking Guide](./concepts/economics/staking.md) 
+ 
+# Support 
+This documentation covers all of the technical aspects of Casper. However, if you can't find what you are looking for, or you simply want to explore the ecosystem, start with the [Support Page](./resources/support.md)
```
Comment (Adrian-Wrona) [inline source/docs/casper/index.md]: Let's not show this section until we have the 'Support' page ready.

```suggestion
```

**Review by Adrian-Wrona — CHANGES_REQUESTED** [review]

File: source/docs/casper/index.md (hunk 1 above)
Comment (ghost) [reply source/docs/casper/index.md:8]: I would prefer to omit the word Documentation altogether then. "Docs" is OK in a URL, but it feels too informal for technical documentation. Do many people read technical documentation on a phone?

File: source/docs/casper/index.md (hunk 5 above)
Comment (ghost) [reply source/docs/casper/index.md:17]: Removed the other heading below instead.

File: source/docs/casper/index.md (hunk 14 above)
Comment (ghost) [reply source/docs/casper/index.md]: This is not part of the Most Visited Pages. it is letting the user know about the other important sectons of the documentaton on a high level.

File: source/docs/casper/concepts/index.md (hunk 8 above)
Comment (ghost) [reply source/docs/casper/concepts/index.md:27]: Couldn't reproduce

File: source/docs/casper/disclaimer.md (hunk 9 above)
Comment (ghost) [reply source/docs/casper/disclaimer.md]: Copy/Paste thing this. Reverting

**Review by Adrian-Wrona — APPROVED** [review]

**Pull request merged**